docs/pages/xy-grid.md

```diff
@@ -18,7 +18,7 @@ The XY grid works very similarly to the standard float grid, but includes a numb
 
 ## Browser support
 
-The XY grid is supported in Chrome, Firefox, Safari 6+, IE10+, iOS 7+, and Android 4.4+. Flexbox is supported in Android 2, but not reliably enough for use with this grid. ([View Flexbox browser support.](https://caniuse.com/#feat=flexbox)) We recommend only using the XY grid on projects that can live with purely cutting-edge browser support.
```

Comment:
Mh, shouldn't it be the https version (newer)?. IE11 is correct.

Reply:
Good catch. There was a minor merge conflict in this file. I took what was in develop. I changed the domain but left out the https.